monai/networks/nets/patchgan_discriminator.py

```diff
@@ -0,0 +1,249 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+from collections.abc import Sequence
+
+import torch
+import torch.nn as nn
+
+from monai.networks.blocks import Convolution
+from monai.networks.layers import Act
+
+
+class MultiScalePatchDiscriminator(nn.Sequential):
+    """
+    Multi-scale Patch-GAN discriminator based on Pix2PixHD:
+
+    High-Resolution Image Synthesis and Semantic Manipulation with Conditional GANs
+    Ting-Chun Wang
```

Comment:
Could mention the paper link here. 
You may check whether this one is the correct link.
"<https://arxiv.org/pdf/1711.11585.pdf>"

Reply:
Done! I added the link to the arxiv abstract page because the pdfs on arxiv can be very slow to load.